src/views/Releases/ListReleases/index.jsx

```diff
@@ -128,22 +155,47 @@ function ListReleases(props) {
             const release = clone(r);
 
             if (sc) {
-              release.scheduledChange = sc;
-              release.scheduledChange.when = new Date(
-                release.scheduledChange.when
-              );
+              release.scheduledChange = buildScheduledChange(sc);
             }
 
-            // todo: set these
-            release.required_signoffs = {};
-
             return release;
           })
         );
       }
     );
   }, []);
 
+  useEffect(() => {
+    const productsReleases = releases.reduce((acc, release) => {
```

Comment:
I think it would be good to make this entire function conditional on whether or not we've already set `requiredSignoffsForProduct`. As things stand now, whenever `releases` changes (eg: when `readOnly` changes for a releases, or a delete is done) we end up redoing all of this work, including a bunch of extra network requests.